index.html

```diff
@@ -1,12 +1,18 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Hello Microverse</title>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+  </head>
+  <body>
+    <header class="full-width">
+      <nav class="main-nav full-width"></nav>
+    </header>
```

Comment:
- [ ] Remove unnecessary code. As you are not using a framework or CSS for this element, and the element itself is empty, you can remove it.